Vwitter/TrackCell.m

```diff
@@ -0,0 +1,36 @@
+//
+//  TrackCell.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/28/22.
+//
+
+#import "TrackCell.h"
+
+@implementation TrackCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
```

Comment:
nit: If the only thing you do in a method is call `super` for that same method, then the entire method can be removed since ObjC will do this by default